api/v1/openapi.yaml

```diff
@@ -2246,8 +2246,6 @@ definitions:
         type: integer
       datapathMode:
         "$ref": "#/definitions/DatapathMode"
-      ipvlanConfiguration:
```

Comment:
I think we'll have to keep this field in the API, just no longer populate it. Otherwise we'de break backwards compatibility.